src/pkg/mcp/deployment_info/deployment_info.go

```diff
@@ -0,0 +1,54 @@
+package deployment_info
```

Comment:
This file looks eerily similar to `src/pkg/cli/getServices.go`. Can we use that one instead?